uglyui/src/router/index.js

```diff
@@ -119,8 +120,9 @@ const router = new VueRouter({
   routes
 });
 
-router.beforeEach((routeTo, routeFrom, next) => {
+router.beforeEach(async (routeTo, routeFrom, next) => {
   NProgress.start();
+  await Auth.isLoaded();
```

Comment:
the problem was on refresh, authentication wasn't completely loaded before the router was invoked, so under the belief that the user wasn't logged in, the router redirected the user to /